website/docs/language/expressions/conditionals.mdx

```diff
@@ -39,6 +39,131 @@ The condition can be any expression that resolves to a boolean value. This will
 usually be an expression that uses the equality, comparison, or logical
 operators.
 
+The following language features are particularly useful when writing condition expressions.
+
+### `contains` Function
+
+Use the built-in function `contains` to test whether a given value is one of a set of predefined valid values.
+
+```hcl
+  condition = contains(["STAGE", "PROD"], var.environment)
+```
+
+### Boolean Operators
+
+Use the boolean operators `&&` (AND), `||` (OR), and `!` (NOT) to combine multiple conditions together.
+
+```hcl
+  condition = var.name != "" && lower(var.name) == var.name
+  ```
```

Comment:
Not sure how picky our Markdown renderer is here, but I believe these leading spaces on the code fence aren't correct.

Reply:
good catch! I fixed this thank you!